handler.go

```diff
@@ -67,10 +76,12 @@ func GetStars(w http.ResponseWriter, req *http.Request) {
 		response.NewErrorResponse(http.StatusInternalServerError, "could not fetch all items").Write(w)
 		return
 	}
+
+	user, _ := getCurrentUser(req)
```

Comment:
I would avoid having the same variable name as a struct, we could call this `u` or `currentUser` or change the struct name to `userInfo` or something.

Reply:
fixed!